lib/src/ui/screen_list/location_model.dart

```diff
@@ -0,0 +1,13 @@
+class LocationModel {
+  final String city;
+  final String street;
+  final String begin1;
+  final String till1;
+  final String begin2;
+  final String till2;
+  final DateTime date;
+
+  // ignore: sort_constructors_first
+  LocationModel(this.city, this.street, this.begin1, this.till1, this.begin2,
+      this.till2, this.date);
```

Comment:
Constructors should be defined before the fields. 
Also it would be nice to set the comma after last parameter